pages/dkp/kommander/2.0/install/networked/index.md

```diff
@@ -66,9 +66,11 @@ If you are installing Kommander on kind, you must know the following:
     ```
 -->
 
-## Install on Konvoy
+## Install Kommander on Konvoy
 
-Before running the commands below make sure that your `kubectl` configuration is pointing to the cluster you want to install Kommander on by setting the `KUBECONFIG` environment variable to the respective kubeconfig file's location.
+Before running the commands below, ensure that your `kubectl` configuration **references the cluster on which you want to install Kommander**, otherwise it will install on the bootstrap cluster. You can do this by setting the `KUBECONFIG` environment variable [to the appropriate kubeconfig file's location][k8s-access-to-clusters].
```

Comment:
```suggestion
Before running these commands, ensure that your `kubectl` configuration **references the cluster where you want to install Kommander**, otherwise it installs on the bootstrap cluster. Do this by setting the `KUBECONFIG` environment variable [to the appropriate kubeconfig file's location][k8s-access-to-clusters].
```

Reply:
Try to avoid using above or below like this, because sometimes we move sections around and then the order of things is off. It's understood that the verbiage here is the lead-in for the commands, so no need to specify which commands.
"will" with a verb is almost always indicator of passive language, so make it active by removing the "will".
Removed "you can", since it is understood - it's not wrong, but take any opportunity you can to reduce how many words needed to say the same thing.